app/controllers/catalog_controller.rb

```diff
@@ -768,7 +771,10 @@ def ot_add
                       :description => "",
                       :content     => "",
                       :type        => ot_type,
-                      :draft       => false}}
+                      :draft       => false,
+                      :manager_id  => ''}}
+    @edit[:new][:available_managers] =
+      ManageIQ::Providers::Openstack::CloudManager.all.collect { |t| [t.name, t.id] }.sort || []
```

Comment:
Could this be something like:

`ManageIQ::Providers::BaseManager.all.select { |manager| manager.supports_vnf_templates? }.collect { |t| [t.name, t.id] }.sort || []`

Would be nicer to avoid referencing `Openstack` directly here.

/cc @durandom 

Thoughts?